eox_audit_model/decorators.py

```diff
@@ -67,30 +73,34 @@ def get(self, request, *args, **kwargs):
     def decorator(func):  # pylint: disable=missing-docstring
         @wraps(func)
         def wrapper(*args, **kwargs):  # pylint: disable=missing-docstring
+            nonlocal data_filter, hidden_fields
             request = args[1]
-            data = request.data if request.data else request.query_params
+            input_parameters = []
+            request_data = request.data if request.data else request.query_params
+            data_filter = [] if data_filter is None else data_filter
+            hidden_fields = [] if hidden_fields is None else hidden_fields
```

Comment:
I wonder why not just change the defaults to `[]` in the keyword args?

Reply:
@mariajgrimaldi that was my first approach and also the easiest, but after doing a little research I found out that using a mutable default value as an argument it's a bad practice
https://docs.quantifiedcode.com/python-anti-patterns/correctness/mutable_default_value_as_argument.html
https://docs.python-guide.org/writing/gotchas/

Reply:
@mariajgrimaldi that would be the easiest approach, but it's not a very good practice to use a mutable default value as an argument  https://docs.quantifiedcode.com/python-anti-patterns/correctness/mutable_default_value_as_argument.html

I know that for now the lists would never be altered inside the function so there is really no risk of having a real problem or bug using these defaults args, but even so, I think the approach would not be very pythonic. What do you think?

Reply:
Using a tuple would be nice, but I'm not a fan of the type mismatch. So fine by me